setup.py

```diff
@@ -6,8 +6,10 @@
 
 
 def read(fname, readlines=False):
-    with open(path.join(path.abspath(path.dirname(__file__)), fname)) as f:
-        return f.readlines() if readlines else f.read()
+    with open(
+        path.join(path.abspath(path.dirname(__file__)), fname), encoding="utf-8"
```

Comment:
Not necessary, none of the documents read while doing setup contains unicode.

Reply:
This trick solved my problem when installing the package from my fork' master branch. Will remove it, do some tests and maybe trigger another pull request